tslearn/shapelets.py

```diff
@@ -317,14 +345,21 @@ def __init__(self, n_shapelets_per_size=None,
         self.weight_regularizer = weight_regularizer
         self.shapelet_length = shapelet_length
         self.total_lengths = total_lengths
+        self.max_size = max_size
+        self.scale = scale
         self.random_state = random_state
 
         if max_iter == 10000:
-            warnings.warn("The default value of max_iter has changed "
+            warnings.warn("The default value for max_iter has changed "
```

Comment:
Perhaps we could remove this warning now since we are at a new major version?

Reply:
Yep, good idea!